pypodcastparser/Item.py

```diff
@@ -103,11 +112,11 @@ def __init__(self, soup):
         self.set_dates_published()
 
     def set_time_published(self):
-        if self.published_date is None:
+        if self.published_date_string is None:
```

Comment:
We're getting a large chunk of unit tests failing on some date formatting logic, wondering if it's tied to this change?  Can you elaborate on why you updated this section?  We might need to update the unit tests to account for this or find a different solution.